nodejs/eks/nodes/nodegroup.ts

```diff
@@ -586,22 +584,22 @@ function createNodeGroup(
     parent: pulumi.ComponentResource,
     provider?: pulumi.ProviderResource,
 ): NodeGroupData {
-    const instanceProfileName = core.apply((c) =>
-        resolveInstanceProfileName(name, args, c, parent),
-    );
+    const validationErrors: pulumi.InputPropertyErrorDetails[] = [];
+
+    const instanceProfileName = core.apply((c) => resolveInstanceProfileName(args, c));
```

Comment:
What happens when you throw inside of an apply? Do we still get the rest of the validation errors unless they depend on `instanceProfileName`? And if we eventually do throw because of other errors do these get surfaced then?